app/src/test/java/com/example/trainingintive/dog_images_feature/presentation/DogImageViewModelTest.kt

```diff
@@ -18,12 +23,9 @@ import java.lang.Exception
 
 class DogImageViewModelTest : RulesForTests() {
 
-    val sampleImageUrl = "https:\\/\\/images.dog.ceo\\/breeds\\/bulldog-french\\/n02108915_9666.jpg"
-    val sampleDog = DogImageUrl(sampleImageUrl)
-    val repository: DogImageRepository = mockk {
-        every { getDogImageUrl() } returns Single.just(sampleDog)
-        every { insert(any()) } returns Completable.complete()
-    }
+    val sampleDog: DogImageUrl = mockk(relaxed = true)
```

Comment:
Avoid creating a lot of models in test file, when it is possible try to create them in function